pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -33,143 +33,12 @@ def get_treeinfo_data(remote):
         treeinfo.load(f=result.path)
         treeinfo_parsed = treeinfo.parsed_sections()
         sha256 = result.artifact_attributes["sha256"]
-        treeinfo_serialized = TreeinfoData(treeinfo_parsed).to_dict(hash=sha256)
+        treeinfo_serialized = TreeinfoData(treeinfo_parsed).to_dict(hash=sha256, filename=namespace)
         break
 
     return treeinfo_serialized
 
 
-def create_treeinfo(distribution_tree):
```

Comment:
Why is this being removed?

Reply:
productmd instroduces some data when parsing a treeinfo,
it sets the header version as 1.2, treats build_timestamp as integer for example.
Creating a treeinfo file could lead to some errors, because we do it relying on parsed data returned from productmd, so I think it is better to stop to create a treeinfo file and just store
the original treeinfo file from the repository link.

Reply:
Ah because kickstarts are immutable. Makes sense to me.